content/docs/identity-providers/azure.mdx

```diff
@@ -1,156 +1,175 @@
 ---
-title: Azure Active Directory
-sidebar_label: Azure AD
+title: Microsoft Entra ID (formerly Azure Active Directory)
+sidebar_label: Microsoft Entra ID (Azure AD)
+description: Learn how to configure Microsoft Entra ID (formerly known as Azure Active Directory) as an identity provider that works with Pomerium Core and Enterprise.
 lang: en-US
-keywords: [azure, active directory, ad, microsoft, identity provider, idp]pagination_prev: null
+keywords:
+  [
+    azure,
+    active directory,
+    ad,
+    entra,
+    entra id,
+    microsoft,
+    identity provider,
+    idp,
+  ]
+pagination_prev: null
 pagination_next: null
 ---
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-This page will help you configure Pomerium to use your Azure Active Directory as an IdP. It assumes you have already [installed Pomerium](/docs/deploy/)
+# Microsoft Entra ID (Azure Active Directory)
+
+Learn how to configure Pomerium to use **Microsoft Entra ID** (formerly Azure Active Directory) as an identity provider.
 
 :::caution
 
 While we do our best to keep our documentation up to date, changes to third-party systems are outside our control. Refer to [Quickstart: Register an application with the Microsoft identity platform](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app) from Microsoft's documentation as needed, or [let us know](https://github.com/pomerium/documentation/issues/new?assignees=&labels=&template=doc-error.md) if we need to re-visit this page.
 
 :::
 
-If you plan on allowing users to log in using a Microsoft Azure Active Directory account, either from your company or from external directories, you must register your application through the Microsoft Azure portal. If you don't have a Microsoft Azure account, you can [sign up](https://azure.microsoft.com/en-us/free) for free.
-
-You can access the Azure management portal from your Microsoft service, or visit [https://portal.azure.com](https://portal.azure.com) and sign in to Azure using the global administrator account used to create the Office 365 organization.
-
-:::tip
+## Before you begin
 
-There is no way to create an application that integrates with Microsoft Azure AD without having **your own** Microsoft Azure AD instance.
-
-:::
+To complete this guide, you must:
 
-If you have an Office 365 account, you can use the account's Azure AD instance instead of creating a new one. To find your Office 365 account's Azure AD instance:
+- Install [Pomerium](/docs/deploy/core)
+- Create a free [Microsoft 365 administration account](https://www.microsoft.com/en-us/microsoft-365/business/office-365-administration) (or have admin privileges with an existing Microsoft 365 account)
```

Comment:
I think you sign up for an Azure account?

https://azure.microsoft.com/en-us/free/

Reply:
You can do that as well. This article explains it a bit better: https://learn.microsoft.com/en-us/microsoft-365/enterprise/azure-integration?view=o365-worldwide